docs/architecture-overview.md

```diff
@@ -152,7 +152,7 @@ Data is deleted after 90 days.
 
 |Role|	Authentication	|Comment|
 | ------------- |:-------------:| -----:|
-|GW to connected Labs (/lab/result Endpoint) | TLS Client Certificate, 2nd factor IP Range | Endpoint is exposed to an other securiy zone
+|GW to connected Labs (/lab/result Endpoint) | TLS Client Certificate, 2nd factor IP Range | Endpoint is exposed to an other security zone
```

Comment:
let's fix this as well...

```suggestion
|GW to connected Labs (/lab/result Endpoint) | TLS Client Certificate, 2nd factor IP Range | Endpoint is exposed to another security zone
```

Reply:
The string 'an other' is vanishingly rare in English. In contrast 'another' is positively pervasive.  However, could it be that 'any other' is meant? See: https://english.stackexchange.com/questions/56435/an-other-vs-another

Reply:
Notwithstanding, 'another', 'an other' or 'any other' is not a typo and probably should not be part of this pull request as this request has the subject 'typo' and no intention to change the meaning, imo.